ci/kind/kind-setup.sh

```diff
@@ -728,5 +734,31 @@ if [[ $ACTION == "create" ]]; then
         echoerr "Only one of '--subnets' and '--extra-networks' can be specified"
         exit 1
     fi
+    if [[ $FLEXIBLE_IPAM == true ]]; then
+        docker network create -d bridge --subnet 192.168.240.0/24 kind
+        # chmod +x ./ci/kind/install-ipset.sh
+        # ./ci/kind/install-ipset.sh 
+        uname -a
+        sudo apt update
```

Comment:
you should be using `docker_run_with_host_net` instead. The script does not need to install any package or run any command as sudo. You will see examples of that in this script to run `iptables` commands.